src/Encamina.Enmarcha.SemanticKernel.Abstractions/Events/MemoryStorageEventExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.ComponentModel;
+
+namespace Encamina.Enmarcha.SemanticKernel.Abstractions.Events;
+
+/// <summary>
+/// <see cref="IMemoryManager.MemoryStorageEvent"/> extensions.
+/// </summary>
+public static class MemoryStorageEventExtensions
+{
+    /// <summary>
+    /// Gets the enum description.
+    /// </summary>
+    /// <param name="value"><see cref="MemoryStorageEventEnum"/> value.</param>
+    /// <returns>Text description.</returns>
+    public static string GetEnumDescription(this MemoryStorageEventEnum value)
```

Comment:
This is OK when the `Description` attribute provides something beyond the name of the enum item. 

For this case, it is better to use `nameof` directly.

Finally, this code would apply to any `enum` type, so it should be more generic and available in a common project of ENMARCHA. And actually, it exists in project `Encamina.Enmarcha.Core` 🫤

It is important to check if ENMARCHA already has something that we might need, and not create ad-hoc code.

Reply:
You are right, it is not necessary in this case